src/components/ChatView.vue

```diff
@@ -161,12 +162,23 @@ export default {
 	mounted() {
 		// Postpone render of NewMessage until application is mounted
 		this.containerId = this.container
+		EventBus.$on('editing-message', () => {
+			this.isEditingMessage = true
+		})
+		EventBus.$on('message-edited', () => {
+			this.isEditingMessage = false
+		})
+	},
+
+	beforeDestroy() {
+		EventBus.$off('editing-message')
```

Comment:
```suggestion
		EventBus.$off('message-editing')
```
let's keep one style for naming

Reply:
Or isn't it better to rely on chatExtrasStore? if some value is defined there, we disable drag-n-drop

Reply:
This is though can be similarly be a getter watcher 👍🏻